src/main/webapp/app/entities/programming-exercise.model.ts

```diff
@@ -68,6 +68,7 @@ export class ProgrammingExerciseBuildConfig {
     public dockerFlags?: string;
     public windFile?: WindFile;
     public testwiseCoverageEnabled?: boolean;
+    public theiaImage?: string;
```

Comment:
If I'm not mistaken, this was moved to build config

Reply:
That's true, forgot about this one line apparently :)